assets/pages/new-contributions-landing/components/ContributionForm.jsx

```diff
@@ -75,6 +75,8 @@ type PropTypes = {|
 
 type FormValueType = string | null;
 
+// We only want to user the user state value if the form state value has not been changed since it was initialised,
```

Comment:
`use the user state value`?

Reply:
We are no longer using the user state to store the details from the form. The user state now doesn't change based on the user's input, and we store the state of the form in `state.page.form.formData`

Reply:
I meant I think there's a spelling mistake 🙃

`user the user state value` -> `use the user state value`?

Reply:
haha oops!

Reply:
can you give it an approve with comment?

Reply:
done